src/engine/source/api/src/test/handlers.cpp

```diff
@@ -302,6 +301,9 @@ api::Handler sessionPost(shared_ptr<Router> router, shared_ptr<Catalog> catalog)
             return genericError<ResponseType>(addRouteError.value().message);
         }
 
+        // Suscribe to output and Trace
+        router->subscribeOutputAndTraces(policyName);
+
```

Comment:
Wouldn't it be better to move this after confirming that the sessions could be created or to unsubscribe in case the session creation fails